=== RUN  TestErrorf
Exception: {
	__self__ : 0x45790, 
	__type__ : {
		gt : named(stdgo._internal.math.bits.Bits_T_errorString.T_errorString,[error: {
	get : <function>
},runtimeError: {
	get : <function>
}],basic(17),false,{
				get : <function>
			})
	}
}
Called from module gt : named(stdgo._internal.math.bits.Bits_T_errorString.T_errorString,[error: {
Called from module get : <function>
Called from module },runtimeError: {
Called from module get : <function>
Called from module }],basic(17),false,{
Called from module get : <function>
Called from module })
Called from module }
Called from module }
Called from .new haxe_ValueException (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 32071 column 17)
Called from haxe_Exception.thrown (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 32001 column 11)
Called from stdgo_AnyInterface.equals (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 31194 column 24)
Called from .stdgo__$internal_reflect_Reflect_$deepEqual_deepEqual (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 45128 column 29)
Called from module at stdgo__$internal_testing_InternalTest._$internal_fmt_$test_Fmt_$test_$testErrorf_testErrorf [as f] (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js:44069:7)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 97052 column 9)
Called from ._$internal_fmt_$dot_$test_Fmt_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 79621 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_fmt.js:122085:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js:122086:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1469 column 14)

-- FAIL: TestErrorf

=== RUN  TestFmtInterface
-- PASS: TestFmtInterface

=== RUN  TestSprintf
Exception: _v.__copy__ is not a function
Called from module TypeError: _v.__copy__ is not a function
Called from stdgo__$internal_reflect_Value_$static_$extension.isValid (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 30413 column 28)
Called from stdgo__$internal_fmt_T_$pp_$static_$extension._printArg (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 27694 column 74)
Called from stdgo__$internal_fmt_T_$pp_$static_$extension._doPrintf (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 25805 column 50)
Called from .stdgo__$internal_fmt_Fmt_$sprintf_sprintf (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 20825 column 48)
Called from module at stdgo__$internal_testing_InternalTest._$internal_fmt_$test_Fmt_$test_$testSprintf_testSprintf [as f] (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js:45188:14)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 97052 column 9)
Called from ._$internal_fmt_$dot_$test_Fmt_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 79621 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_fmt.js:122085:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js:122086:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1469 column 14)

-- FAIL: TestSprintf

=== RUN  TestComplexFormatting
-- PASS: TestComplexFormatting

=== RUN  TestReorder
-- PASS: TestReorder

=== RUN  TestCountMallocs
<invalid reflect.Value>
Exception: __skip__
Called from stdgo__$internal_testing_T_$common_$static_$extension.skipNow (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 97601 column 23)
Called from stdgo__$internal_testing_T_$common_$static_$extension.skip (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 97631 column 56)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_fmt.js:97236:58
Called from module at stdgo__$internal_testing_InternalTest._$internal_fmt_$test_Fmt_$test_$testCountMallocs_testCountMallocs [as f] (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js:45621:18)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 97052 column 9)
Called from ._$internal_fmt_$dot_$test_Fmt_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 79621 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_fmt.js:122085:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js:122086:3)
-- SKIP: TestCountMallocs

=== RUN  TestFlagParser
-- PASS: TestFlagParser

=== RUN  TestStructPrinter
Sprintf(<invalid reflect.Value>, s) = %!q(MISSING), want %!q(MISSING)Sprintf(<invalid reflect.Value>, &s) = %!q(MISSING), want %!q(MISSING)
-- FAIL: TestStructPrinter

=== RUN  TestSlicePrinter
-- PASS: TestSlicePrinter

=== RUN  TestMapPrinter
-- PASS: TestMapPrinter

=== RUN  TestEmptyMap
-- PASS: TestEmptyMap

=== RUN  TestBlank
-- PASS: TestBlank

=== RUN  TestBlankln
-- PASS: TestBlankln

=== RUN  TestFormatterPrintln
-- PASS: TestFormatterPrintln

=== RUN  TestWidthAndPrecision
-- PASS: TestWidthAndPrecision

=== RUN  TestPanics
-- SKIP: testPanics
-- PASS: TestPanics

=== RUN  TestBadVerbRecursion
-- PASS: TestBadVerbRecursion

=== RUN  TestIsSpace
-- PASS: TestIsSpace

=== RUN  TestNilDoesNotBecomeTyped
expected:
	<invalid reflect.Value>
got:
	%!q(MISSING)
-- FAIL: TestNilDoesNotBecomeTyped

=== RUN  TestFormatterFlags
Sprintf(<invalid reflect.Value>, %!T(MISSING)) = %!q(MISSING), want %!q(MISSING)
-- FAIL: TestFormatterFlags

=== RUN  TestParsenum
-- PASS: TestParsenum

=== RUN  TestAppendf
-- PASS: TestAppendf

=== RUN  TestAppend
-- PASS: TestAppend

=== RUN  TestAppendln
-- PASS: TestAppendln

=== RUN  TestScan
-- PASS: TestScan

=== RUN  TestScanln
-- PASS: TestScanln

=== RUN  TestScanf
Exception: array out of bounds, index: 0 length: 0
Called from stdgo__$internal_fmt_T_$ss_$static_$extension._doScanf (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 48102 column 25)
Called from .stdgo__$internal_fmt_Fmt_$fscanf_fscanf (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 53594 column 62)
Called from .stdgo__$internal_fmt_Fmt_$sscanf_sscanf (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 53527 column 56)
Called from module at stdgo__$internal_testing_InternalTest._$internal_fmt_$test_Fmt_$test_$testScanf_testScanf [as f] (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js:53490:18)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 97052 column 9)
Called from ._$internal_fmt_$dot_$test_Fmt_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 79621 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_fmt.js:122085:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js:122086:3)

-- FAIL: TestScanf

=== RUN  TestScanOverflow
Exception: array out of bounds, index: 0 length: 0
Called from .stdgo__$internal_regexp_syntax_Syntax_$_$parse__parse (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 54835 column 25)
Called from .stdgo__$internal_regexp_syntax_Syntax_$parse_parse (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 53821 column 9)
Called from .stdgo__$internal_regexp_Regexp_$_$compile__compile (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 53655 column 16)
Called from .stdgo__$internal_regexp_Regexp_$compile_compile (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 53652 column 9)
Called from .stdgo__$internal_regexp_Regexp_$mustCompile_mustCompile (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 53637 column 16)
Called from module at stdgo__$internal_testing_InternalTest._$internal_fmt_$test_Fmt_$test_$testScanOverflow_testScanOverflow [as f] (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js:53616:12)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 97052 column 9)
Called from ._$internal_fmt_$dot_$test_Fmt_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 79621 column 59)

-- FAIL: TestScanOverflow

=== RUN  TestNaN
Exception: array out of bounds, index: 0 length: 0
Called from stdgo__$internal_fmt_T_$ss_$static_$extension._doScan (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 48273 column 25)
Called from .stdgo__$internal_fmt_Fmt_$fscan_fscan (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 47438 column 62)
Called from ._$internal_fmt_$test_Fmt_$test_$_$verifyNaN__verifyNaN (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 71915 column 16)
Called from module at stdgo__$internal_testing_InternalTest._$internal_fmt_$test_Fmt_$test_$testNaN_testNaN [as f] (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js:71779:3)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 97052 column 9)
Called from ._$internal_fmt_$dot_$test_Fmt_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 79621 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_fmt.js:122085:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js:122086:3)

-- FAIL: TestNaN

=== RUN  TestInf
Exception: array out of bounds, index: 0 length: 0
Called from stdgo__$internal_fmt_T_$ss_$static_$extension._doScan (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 48273 column 25)
Called from .stdgo__$internal_fmt_Fmt_$fscan_fscan (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 47438 column 62)
Called from ._$internal_fmt_$test_Fmt_$test_$_$verifyInf__verifyInf (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 72083 column 16)
Called from module at stdgo__$internal_testing_InternalTest._$internal_fmt_$test_Fmt_$test_$testInf_testInf [as f] (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js:71947:3)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 97052 column 9)
Called from ._$internal_fmt_$dot_$test_Fmt_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 79621 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_fmt.js:122085:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js:122086:3)

-- FAIL: TestInf

=== RUN  TestScanfMulti
-- PASS: TestScanfMulti

=== RUN  TestScanMultiple
Exception: array out of bounds, index: 0 length: 0
Called from stdgo__$internal_fmt_T_$ss_$static_$extension._doScan (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 48273 column 25)
Called from .stdgo__$internal_fmt_Fmt_$fscan_fscan (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 47438 column 62)
Called from .stdgo__$internal_fmt_Fmt_$sscan_sscan (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 66268 column 54)
Called from module at stdgo__$internal_testing_InternalTest._$internal_fmt_$test_Fmt_$test_$testScanMultiple_testScanMultiple [as f] (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js:72272:16)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 97052 column 9)
Called from ._$internal_fmt_$dot_$test_Fmt_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 79621 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_fmt.js:122085:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js:122086:3)

-- FAIL: TestScanMultiple

=== RUN  TestScanEmpty
<invalid reflect.Value>
<invalid reflect.Value>

-- FAIL: TestScanEmpty

=== RUN  TestScanNotPointer
Exception: _v.set is not a function
Called from stdgo__$internal_fmt_T_$ss_$static_$extension._doScan (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 48273 column 25)
Called from .stdgo__$internal_fmt_Fmt_$fscan_fscan (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 47438 column 62)
Called from module at stdgo__$internal_testing_InternalTest._$internal_fmt_$test_Fmt_$test_$testScanNotPointer_testScanNotPointer [as f] (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js:72438:16)
Called from stdgo__$internal_testing_M_$static_$extension.run (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 97052 column 9)
Called from ._$internal_fmt_$dot_$test_Fmt_$dot_$test_main (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js line 79621 column 59)
Called from module at /home/runner/work/go2hx/go2hx/golibs/std_fmt.js:122085:1
Called from module at Object.<anonymous> (/home/runner/work/go2hx/go2hx/golibs/std_fmt.js:122086:3)
Called from Module._compile (node:internal/modules/cjs/loader line 1469 column 14)

-- FAIL: TestScanNotPointer

=== RUN  TestScanlnNoNewline
<invalid reflect.Value>

-- FAIL: TestScanlnNoNewline

=== RUN  TestScanlnWithMiddleNewline
<invalid reflect.Value>

-- FAIL: TestScanlnWithMiddleNewline

=== RUN  TestEOF
<invalid reflect.Value>

-- FAIL: TestEOF

=== RUN  TestEOFAtEndOfInput
Sscanf expected EOF; got <invalid reflect.Value>Sscan expected EOF; got <invalid reflect.Value>Sscan expected EOF; got <invalid reflect.Value>
-- FAIL: TestEOFAtEndOfInput

=== RUN  TestEOFAllTypes
#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on empty string: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on trailing blanks: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on empty string: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on trailing blanks: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on empty string: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on trailing blanks: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on empty string: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on trailing blanks: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on empty string: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on trailing blanks: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on empty string: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on trailing blanks: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on empty string: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on trailing blanks: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on empty string: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on trailing blanks: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on empty string: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on trailing blanks: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on empty string: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on trailing blanks: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on empty string: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on trailing blanks: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on empty string: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on trailing blanks: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on empty string: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on trailing blanks: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on empty string: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on trailing blanks: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on empty string: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on trailing blanks: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on empty string: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on trailing blanks: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on empty string: %!s(MISSING)#<invalid reflect.Value>: %!s(MISSING) %!T(MISSING) not eof on trailing blanks: %!s(MISSING)
-- FAIL: TestEOFAllTypes

=== RUN  TestUnreadRuneWithBufio
-- PASS: TestUnreadRuneWithBufio

=== RUN  TestMultiLine
<invalid reflect.Value>

-- FAIL: TestMultiLine

=== RUN  TestLineByLineFscanf
-- PASS: TestLineByLineFscanf

=== RUN  TestScanStateCount
-- PASS: TestScanStateCount

=== RUN  TestScanInts
-- PASS: TestScanInts

=== RUN  TestHexBytes
<invalid reflect.Value>: bad length %!d(MISSING)<invalid reflect.Value>: bad length %!d(MISSING)<invalid reflect.Value>: bad length %!d(MISSING)<invalid reflect.Value>: bad length %!d(MISSING)<invalid reflect.Value>: bad length %!d(MISSING)<invalid reflect.Value>: bad length %!d(MISSING)odd count: got count, err = <invalid reflect.Value>, %!v(MISSING); expected 0, error
-- FAIL: TestHexBytes

=== RUN  TestScanNewlinesAreSpaces
-- PASS: TestScanNewlinesAreSpaces

=== RUN  TestScanlnNewlinesTerminate
<invalid reflect.Value>: expected error; got none<invalid reflect.Value>: expected error; got none
-- FAIL: TestScanlnNewlinesTerminate

=== RUN  TestScanfNewlineMatchFormat
<invalid reflect.Value>: expected error; got none<invalid reflect.Value>: expected error; got none<invalid reflect.Value>: expected error; got none<invalid reflect.Value>: expected error; got none<invalid reflect.Value>: expected error; got none<invalid reflect.Value>: expected error; got none<invalid reflect.Value>: expected error; got none<invalid reflect.Value>: expected error; got none<invalid reflect.Value>: expected error; got none
-- FAIL: TestScanfNewlineMatchFormat

=== RUN  TestHexByte
-- PASS: TestHexByte

=== RUN  TestFormatString
-- PASS: TestFormatString

=== RUN  TestStringer
-- PASS: TestStringer

